<?xml version="1.0"?>
<launch>
    <arg name="sim" value="true"/>

    <include file="$(find docking_sim)/launch/link.launch"/>
    
    <node pkg="rap_controller" type="rap_controller.py" name="rap_controller_car1" output='screen'>
        <param name="robot_name" value="car1"/>
        <param name="role" value="leader"/>
        <param name="sim" value="$(arg sim)"/>
        <remap from="/car1/cmd_vel" to="/cmd_vel_car1"/>
        <remap from="/car1/naive_cmd" to="/cmd_vel"/>
    </node>
    <node pkg="rap_controller" type="rap_controller.py" name="rap_controller_car2" output='screen'>
        <param name="robot_name" value="car2"/>
        <param name="role" value="follower"/>
        <param name="sim" value="$(arg sim)"/>
        <remap from="/car2/cmd_vel" to="/cmd_vel_car2"/>
        <remap from="/car2/naive_cmd" to="/cmd_vel"/>
    </node>
    <!--node pkg="rviz" type="rviz" name="rviz" args="-d $(find ros_pkg_example)/rviz/default.rviz" if="$(arg use_rviz)" /-->
</launch>
